frontend/packages/console-app/src/components/modals/restore-pvc/restore-pvc-modal.tsx

```diff
@@ -144,9 +137,9 @@ const RestorePVCModal = withHandlePromise<RestorePVCModalProps>(
       );
     };
     return (
-      <div className="modal-content modal-content--no-inner-scroll">
-        <ModalTitle>{t('console-app~Restore as new PVC')}</ModalTitle>
-        <Form onSubmit={submit} name="form">
+      <form onSubmit={submit} name="form">
+        <div className="modal-content modal-content--no-inner-scroll">
```

Comment:
It looks like this `div` should be changed to the `form` element (keeping the classnames) based on @rhamilto's comment.

Reply:
mm i thought both are same, anyway let me try without div

Reply:
It may render the same (I did not test), but the `<div>` is unnecessary, so it's better without.

Reply:
ack